contrib/bazel/patches/cbuild_flags.patch

```diff
@@ -0,0 +1,53 @@
+diff --color -ruN bazel-7.3.0/tools/cpp/unix_cc_toolchain_config.bzl bazel-7.3.0-patched/tools/cpp/unix_cc_toolchain_config.bzl
+--- bazel-7.3.0/tools/cpp/unix_cc_toolchain_config.bzl	1980-01-01 00:00:00.000000000 +0000
++++ bazel-7.3.0-patched/tools/cpp/unix_cc_toolchain_config.bzl	2024-08-14 13:56:00.985516135 +0000
+@@ -1426,6 +1426,41 @@
+         ],
+     )
+ 
++    cbuild_flags_feature = feature(
++        name = "cbuild_flags",
++        enabled = True,
++        flag_sets = [
++            flag_set(
++                actions = all_compile_actions,
++                flag_groups = [flag_group(flags = [
++                    "-D_LARGEFILE64_SOURCE",
```

Comment:
we're very unlikely to merge this with the problematic copying of the flags from cbuild  (will easily go out of sync, etc.)

i briefly looked up making the build respect our env vars and found https://github.com/bazelbuild/bazel/issues/1258, which was opened in 2016 and closed by a stalebot, so not very encouraging

Reply:
Fair enough

Reply:
it could probably be done dynamically but not sure what that would look like

Reply:
feel free to give it a try anyway :)